### PR TITLE
Fix replication warnings

### PR DESCRIPTION
**Pull request by mwiencek**
These commits were part of my dbmirror2 branch but got lost somehow (maybe I never pushed them?).

### CODE REVIEW

**Review by reosarevok — APPROVED** [review]
Seems sensible enough.

File: admin/RunExport [hunk 1]
```diff
@@ -36,24 +36,20 @@ fi
 # export then $EXPORT_STAMP is empty, so that argument is ignored.
 
 mkdir -m "$BACKUP_DIR_MODE" -p \
-    "$BACKUP_DIR"/fullexport/ \
```
[the remark this answers is not in the document]
Comment (mwiencek) [reply admin/RunExport:39]: That's correct.  Listing out each directory shouldn't be needed with `p`.

**Review by yvanzo — APPROVED** [review]
LGTMBDNT :ship:

By the way, I never knew that MB used to produce bundled packets. Do you know what it was used for and why it has been discontinued? We are better off producing useless packets anyway.

**Comment by mwiencek** [comment]
> By the way, I never knew that MB used to produce bundled packets. Do you know what it was used for and why it has been discontinued? We are better off producing useless packets anyway.

The idea behind them is documented in MBS-5272, though later Rob gave the OK to remove them (maybe they weren't widely used in practice).

**Pull request merged**